Project/Source Files/Screens/Controllers/HomeViewController.swift

```diff
@@ -7,9 +7,20 @@ import UIKit
 
 class HomeViewController: UIViewController {
     
+    // MARK: - Constants
+    
+    private struct Constants {
+        static let homeString = "Home"
```

Comment:
If a thing is used only one time, then ther e is no reason do put in in constants struct. Delete it